docs/modules/operation/pages/deep-dive/flows/introduction.adoc

```diff
@@ -2,44 +2,88 @@
-* Processing xref:operation:deep-dive/flows/sentinel/sentinel.adoc#flows-scaling[large volume of flows data] (add Sentinel to scale)
-* Issues with flows at scale and queries taking too long (add the OpenNMS https://github.com/OpenNMS/nephron[streaming analytics tool])
-
-.Flow integration overview
-image::flows/flow_integration_overview.png[width=70%]
+* xref:deep-dive/flows/basic.adoc#flows-basic[Basic setup].
+* Collect flows data in a xref:deep-dive/flows/distributed.adoc#flows-remote[distributed/remote network] (add a Minion).
+* Process a xref:deep-dive/flows/sentinel/sentinel.adoc#flows-scaling[large volume of flows data] (add Sentinel to scale).
+* Issues with flows at scale and queries taking too long (add the OpenNMS https://github.com/OpenNMS/nephron[streaming analytics tool]).
 
 == How it works
 
-At a high level, with a xref:operation:deep-dive/flows/basic.adoc#flows-basic[basic setup], OpenNMS processes flows as follows:
+At a high level, with a xref:deep-dive/flows/basic.adoc#flows-basic[basic setup], {page-component-title} processes flows as follows:
 
-* <<deep-dive/telemetryd/introduction.adoc#ga-telemetryd, Telemetryd>> receives and decodes flows on {page-component-title}.
+* <<deep-dive/telemetryd/introduction.adoc#ga-telemetryd, Telemetryd>> receives and decodes flows.
 * Telemetryd adapters convert the flows to a canonical flow model.
 * Flows are enriched:
 ** The <<deep-dive/flows/classification-engine.adoc#ga-flow-support-classification-engine, classification engine>> tags flows and groups them under a name based on a set of rules.
-** Metadata related to associated nodes (such as IDs and categories) are also added to the flows.
+** Metadata related to associated nodes (such as IDs and categories) are added to the flows.
 * Enriched flows are persisted in Elasticsearch and/or forwarded to Kafka.
-* You can use the OpenNMS streaming analytics tool to aggregate flows and output aggregates to Elasticsearch, Cortex, or Kafka
-* The REST API supports generating both summaries and time series data from the flows or flow aggregates stored in Elasticsearch.
-* Use OpenNMS Helm to visualize flows and/or flow aggregates:
-** Use the "Flow Deep Dive" dashboard with an OpenNMS Flow datasource to visualize flows and flow aggregates stored in Elasticsearch.
-** Use the "Cortex Flow Deep Dive" dashboard with a Prometheus datasource to access flow aggregates stored in Cortex.
+* (Optional) The OpenNMS streaming analytics tool aggregates flows and outputs them to Elasticsearch, Cortex, or Kafka.
+* (Optional) The REST API generates summaries and time series data from the flows or aggregates stored on Elasticsearch.
+* OpenNMS Helm visualizes flows and aggregates:
+** The Flow Deep Dive dashboard visualizes flows and aggregates that are stored in Elasticsearch using a flows datasource.
+** The Cortex Flow Deep Dive dashboard visualizes aggregates that are stored in Cortex using a Prometheus datasource.
```

Comment:
Same question as line 42. Which is using the datasource?

```suggestion
** The Cortex Flow Deep Dive dashboard uses a Prometheus datasource to visualize aggregates that are stored in Cortex.
```

If the former:

The Cortex Flow Deep Dive dashboard visualizes aggregates that are stored in Cortex from a Prometheus datasource.